node/src/neighborhood/neighborhood_database.rs

```diff
@@ -50,15 +51,20 @@ impl NeighborhoodDatabase {
             logger: Logger::new("NeighborhoodDatabase"),
         };
 
-        let mut node_record = NodeRecord::new(
-            public_key,
+        let location = if let Some(node_addr) = neighborhood_mode.node_addr_opt() {
+            get_node_location(Some(node_addr.ip_addr))
```

Comment:
in case, we want default behaviour of Node without specified IP to be working with ZZ country_code I am using the default() impl for NodeAddr, so it gives me ZZ.